manifests/init.pp

```diff
@@ -287,6 +287,16 @@
     before                         => Class['foreman_proxy::plugin::pulp'],
   }
 
+  if ! $pulpcore_mirror {
```

Comment:
the mirrors (aka proxies) don't have the certs required

Reply:
I think this is technically a bit more accurate?
```suggestion
  if $shared_with_foreman_vhost {
```

Technically it's the same right now, but the idea behind it is that it could end up deploying just a content proxy without Foreman but still is a non-mirror. Essentially you could end up with `foreman.example.com` (with Katello, without Pulp) and `pulp.example.com` (primary content storage). Then you could also have `mirror.location.example.com` which would be a local mirror.

Reply:
Possible, yeah. I must admit I did not exactly understand the difference from the code. Your explanation makes sense :)

Reply:
I captured my thoughts in https://github.com/theforeman/puppet-foreman_proxy_content/issues/407